packages/expo-dev-launcher/ios/DevLauncherNetworkInterceptor.swift

```diff
@@ -0,0 +1,130 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+import React
+
+#if DEBUG && EX_DEV_CLIENT_NETWORK_INSPECTOR
+
+@objc(EXDevLauncherNetworkInterceptor)
+public final class DevLauncherNetworkInterceptor: NSObject, ExpoRequestCdpLoggerDelegate {
+  private static var isHookSetuped = false
```

Comment:
Hmm I think "setuped" is an incorrect past form. Generally "setup" is not a verb but a noun, so `isHookSetUp` would be more correct but I would just use `isHookInstalled`.
Moreover, moving this var to the top-level scope might be a good idea too.